p2p/p2p_test.go

```diff
@@ -27,13 +28,14 @@ import (
 
 	"github.com/ChainSafe/gossamer/common"
 	"github.com/ChainSafe/gossamer/common/optional"
+	net "github.com/libp2p/go-libp2p-core/network"
 	peer "github.com/libp2p/go-libp2p-core/peer"
 	ps "github.com/libp2p/go-libp2p-core/peerstore"
 	ma "github.com/multiformats/go-multiaddr"
 )
 
-func startNewService(t *testing.T, cfg *Config, msgChan chan []byte) *Service {
-	node, err := NewService(cfg, msgChan)
+func startNewService(t *testing.T, cfg *Config, msgChan chan []byte, msgRecChan chan BlockAnnounceMessage) *Service {
```

Comment:
Again, not necessary, but we might want to update `msgChan` to `msgSendChan` here and below for clarity/consistency.